### ISSUE TITLE
result table not rendering

### ISSUE THREAD

**Issue by vpnagraj** [databio/LOLAweb#44]
currently the JS datatable at the bottom of the results view renders locally but not on dev:

http://dev.lolaweb.databio.org/?key=VWQN3ZC5HFD92EK

Chrome developer tools is reporting the following error for `datatables.js`:

```
Failed to load resource: the server responded with a status of 404 (Not Found)
```

can't remember if i saw this before the navbarpage change or not ... but i'm wondering if this is an issue with websockets being disabled?

**Comment by nsheff** [comment]
The 'download plot' buttons are gone as well.

**Comment by nmagee** [comment]
Pete, I see something that might be related here -- when I view the source of the page I'm seeing two `datatables.js` files referenced in `<script>` tags. One is **datatables-bindning-0.4** and the other **datatables-bindning-0.2**.

Perhaps the redundant file src is quashing the js from loading, or ?

**Comment by nmagee** [comment]
Okay yes that's got to be it -- the latter reference to 0.2 does not exist. So the browser is trying to retrieve the last-mentioned JS file and that's what's giving us a 404 on that.

**Comment by vpnagraj** [comment]
hmmmm okay ... well i'm not managing how any of those js files are referenced (that should be done by shiny) so maybe an issue with the `DT` package? 

either way this helps, i'll look into it

**Comment by nmagee** [comment]
Okay cool. Yeah, I wonder if two libraries are referencing that and one is relying on a stale version.

**Comment by vpnagraj** [comment]
fixed this in dc22e19

looks like we had a redundant `<head>` in the about.html and the app was calling the `datatables` js library twice

d'oh!
